### PR TITLE
Move @react-native-firebase/utils into /app

### PR DESCRIPTION
**Pull request by IljaDaderko**

### CODE REVIEW

File: docs/utils/quick-start.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ description: Getting started with the Utils package in React Native Firebase
 Install this module with Yarn:
 
 ```bash
-yarn add @react-native-firebase/utils
+yarn add @react-native-firebase/app
```
Comment (Ehesp) [inline docs/utils/quick-start.md]: I guess we can remove this directory now, or at least migrate any docs into the app directory?

File: docs/utils/quick-start.md [hunk 2]
```diff
@@ -1,49 +0,0 @@
...
-import analytics from '@react-native-firebase/analytics';
-
-async function bootstrap() {
-  if (utils().isRunningInTestLab) {
-    await analytics().setAnalyticsCollectionEnabled(false);
-  }
-}
-```
-
-### Access device file paths
```
Comment (Salakar) [inline docs/utils/quick-start.md]: This section should be kept, shouldn't it? Don't see it documented elsewhere

File: docs/app/quick-start.md [hunk 3]
```diff
@@ -57,4 +57,33 @@ import auth, { firebase } from '@react-native-firebase/auth';
...
+import { utils } from '@react-native-firebase/app';
+import analytics from '@react-native-firebase/analytics';
+
+async function bootstrap() {
+  if (utils().isRunningInTestLab) {
+    await analytics().setAnalyticsCollectionEnabled(false);
+  }
+}
+```
+
```
Comment (Salakar) [inline docs/app/quick-start.md]:
````suggestion

### Access device file paths

Some modules require access to your local device filesystem (such as Storage & ML Kit Vision). The utils module provides paths to common device directory locations.

```js
import firebase from '@react-native-firebase/app';
// Access the device pictures directory
const picturesDir = firebase.utils.FilePath.PICTURES_DIRECTORY;
```
````

File: docs/utils/quick-start.md (hunk 2 above)
Comment (Salakar) [reply docs/utils/quick-start.md]: Added a suggested change for it

File: packages/utils/e2e/utils.e2e.js [hunk 4]
```diff
@@ -1,32 +0,0 @@
-/*
```
Comment (Salakar) [inline packages/utils/e2e/utils.e2e.js]: Need to keep this file but can move it into `packages/app/e2e` without issues.

File: packages/utils/e2e/utilsStatics.e2e.js [hunk 5]
```diff
@@ -1,68 +0,0 @@
-/*
```
Comment (Salakar) [inline packages/utils/e2e/utilsStatics.e2e.js]: Need to keep this file but can move it into `packages/app/e2e` without issues.

File: packages/utils/type-test.ts [hunk 6]
```diff
@@ -1,26 +0,0 @@
-import firebase from '@react-native-firebase/app';
```
Comment (Salakar) [inline packages/utils/type-test.ts]: Could do with keeping this file but can move it into the root of `packages/app` without issues.

These `type-test` files are here just to validate the module usages work as expected when TSC runs over the project.

Could do with adding some lines on this as well that use some of the `FilePath` statics as well, just to make sure.

**Pull request merged**